tests/src/test/scala/system/basic/WskCliRuleTests.scala

```diff
@@ -22,7 +22,35 @@ import org.scalatest.junit.JUnitRunner
 
 import common.Wsk
 
+import spray.json._
+import spray.json.DefaultJsonProtocol._
+
 @RunWith(classOf[JUnitRunner])
 class WskCliRuleTests extends WskRuleTests {
   override val wsk: Wsk = new Wsk
+
+  it should "preserve rule status when a rule is updated" in withAssetCleaner(wskprops) { (wp, assetHelper) =>
```

Comment:
Does this test need to different from the one in https://github.com/apache/incubator-openwhisk/pull/3503/files?

Reply:
see https://github.com/apache/incubator-openwhisk/pull/3503/files#r178551945

I can merge the tests w/ some logic if we find that is more appropriate

Reply:
I have merged them, see https://github.com/apache/incubator-openwhisk/pull/3503/files#diff-907d34bcf62c94014993d5450a69ba23R106